packages/central-server/app/sync/CentralSyncManager.js

```diff
@@ -581,6 +581,7 @@ export class CentralSyncManager {
     try {
       // commit the changes to the db
       const persistedAtSyncTick = await sequelize.transaction(async () => {
+        await sequelize.setTransactionVar(AUDIT_PAUSE_KEY, true);
```

Comment:
This is random enough amongst the rest of the code in her to deserve a comment imo, adds coupling to a completely different system that you haven't otherwise had to think about while reading through CentralSyncManager

Reply:
@passcod Are you able to add a comment in here and then we can go ahead with the merge?

Reply:
@tcodling can you please add the comment? I know @passcod is trying to get us as much assurance on sync testing of auditability today as possible, so be great to leave them focused on that

Reply:
Ill try my best just I am totally out of the loop what is going on here 😆

Reply:
Something like this, although I don't know if the comment about mobile is true @dannash100??

```suggestion
        // Don't produce change logs for changes made through sync
        // Auditable change logs are generated on facility servers and will sync in to central
        // For mobile, we generate change logs on central through a separate mechanism
        await sequelize.setTransactionVar(AUDIT_PAUSE_KEY, true);
```

Reply:
Yeah thats true re mobile just not currently implemented

Reply:
Comment looks good to me @tcodling